libs/xb/op.ml

```diff
@@ -31,13 +31,13 @@ let operation_c_mapping =
            Transaction_end; Introduce; Release;
            Getdomainpath; Write; Mkdir; Rm;
            Setperms; Watchevent; Error; Isintroduced;
-           Resume; Set_target |]
+           Resume; Set_target; |]
```

Comment:
Is this semicolon necessary? Just below it the list you modified does not have one.